package.json

```diff
@@ -24,7 +24,7 @@
   "author": "snyk.io",
   "license": "Apache-2.0",
   "dependencies": {
-    "@snyk/dep-graph": "1.19.4",
+    "@snyk/dep-graph": "^1.21.0",
```

Comment:
are we ok with `^`?

Reply:
Yep. I trust how disciplined we are with `@snyk/dep-graph` releases.